fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/provider/impl/FileProvider.java

```diff
@@ -212,4 +213,35 @@ public void writeResources(String mediaType, List<ReadResultDTO> dtos) throws Ex
 
         chunkData.setLastWrittenPageNum(chunkData.getPageNum());
     }
+
+    @Override
+    public void pushEndOfJobOperationOutcomes(ByteArrayOutputStream baos, String folder, String fileName)
+            throws FHIRException {
+            String base = configuration.getBaseFileLocation(source);
```

Comment:
minor formatting thing:  unindent method body

Reply:
the method body is still double-indented, but I'm fine with it for now if everything else checks out

Reply:
hum... it doesn't show that locally on my machine

Reply:
I think your view is out of date.  It shows correctly here.